docs/workflows-config.md

```diff
@@ -0,0 +1,116 @@
+---
+id: workflows-config
+title: Workflows configuration
+sidebar_label: Workflows configuration
+---
+
+This page provides documentation for `buildbuddy.yaml`, which can be placed
+at the root of your git repo to configure BuildBuddy [workflow](workflows) execution.
+
+_NOTE: This file only takes effect after you
```

Comment:
Not sure how this renders currently, but you can use one of these Note: "admonitions" https://docusaurus.io/docs/markdown-features/admonitions

Reply:
Oh nice -- used the "note" one and it looks pretty nice.